utils/CMakeLists.txt

```diff
@@ -111,6 +121,7 @@ target_include_directories(portable-model-info PRIVATE "${PROJECT_SOURCE_DIR}/cp
 set_target_properties(portable-model-info PROPERTIES OUTPUT_NAME ${PORTABLE_MODEL_INFO})
 
 add_executable(simulator-model simulator-model.cpp)
+target_include_directories(simulator-model PRIVATE "${PROJECT_SOURCE_DIR}/cpp/src")
```

Comment:
Why is this necessary?

Reply:
I'm glad you asked. In fact, it's no longer needed. I introduced it, because I had started to replace some of the path handling code in `simulator-model.cpp` with our new `KIM::FILESYSTEM::Path` class in order to solve a problem, which I thought was caused by these kinds of statements:
```
std::string const filePath = *dirName + "/" + *name;
```
I thought it would be necessary to replace them with a more portable solution and made sure that the code has access to `KIM_FilesystemPath.hpp`. 

However, I've reversed these code changes after I realized that the problem was not related to file path handling. Instead, it was caused by the different line endings on the Windows platform. That's why you see a different change to `simulator-model.cpp` in my updated pull request: The `smspec` file must be opened in binary mode ("rb") to make sure the number of bytes read from the file exactly matches the file's size. 

I will revert the change made to the CMakeLists.txt file.